_posts/2024-05-14-ASUworkshop.md

```diff
@@ -0,0 +1,171 @@
+title: "Save the Date: June 24th-25th, 2024 - in-person workshop at Arizona State University"
+---
+
+<img
+src="/public/images/ASUworkshop_cactus.png"
+title="MDAnalysis ASU workshop 2024" alt="MDAnalysis Workshop June 24-25th, 2024 at Arizona State University"
+style="float: right; height: 20em; " />
+
+<img
+src="/public/images/MolSSI_Logo.png"
+title="MolSSI logo" alt="MolSSI logo"
+style="float: right; height: 5em; " />
+
+MDAnalysis, in conjunction with the [Molecular Sciences Software Institute][]
+(MolSSI) and with the support of the [Chan-Zuckerberg Inititative][CZI] and the
+[Center for Biological Physics][CBP], is holding a **2-day** workshop on **June
+24th-25th** 2024 at **Arizona State University** in Tempe, Arizona, USA.
+
+This two-day workshop is intended to transform attendees from users to
+developers and will cover an introduction to MDAnalysis, software best
+practices, and guide participants through the creation of their own [MDAKit][mdakits].
+
+### Workshop outline
+
+MDAnalysis developers and experienced MolSSI instructors will teach
+participants how to build their own MDAnalysis-based Python packages by guiding
+them through the development of their own MDAKits. Specifically, the workshop
+will include 3 modules: (1) an introduction to using and writing custom
+analyses in MDAnalysis; (2) an overview of software development and maintenance
+best practices; and (3) an interactive hackathon session where participants
+write their own MDAKits.
+
+The program will run from the morning of Monday, June 24th until the early
+evening of Tuesday June 25th. 
+
+Parts of the workshop will be held in a hybrid mode for those unable to attend
+in-person. Registration for the hybrid components is required. Recordings of
+hybrid sessions will be made available after the workshop.
+
+Breakfast and lunch will will be provided together with tea/coffee during
+breaks.
+
+
+
+### Registration
+
+Attendance at this workshop will be *free* but limited to 40 in-person
+participants.
+
+A number of **bursaries** will be available to assist with travel and
+accommodation costs for participants from outside the Phoenix area. Bursaries
+(up to *$600 in costs for US domestic travel*) and *up to 3 nights of hotel
+accommodation* (single room within 5 minutes of the workshop venue) will be
+awarded.
+
+#### Application *with bursary request* (non-local attendees)
+
+Register by **May 27, 2024** (midnight anywhere on Earth).
+
+We especially welcome your application if you consider yourself being part of
+an under-represented demographic in the computational molecular sciences.
+
+Bursaries are reserved to enable the attendance of participants from outside
+the greater Phoenix area.
+
+Note that airfare for travel awardees will have to be *booked by June 3 through
+our travel agency* so as an awardee you will have to be able to
+respond to our communications (via email) immediately and act quickly.
+
+Bursary recipients will be notified by May 29. 
```

Comment:
@IAlibay @fiona-naughton @jennaswa @armcdona we need to tell anyone who applied for a bursary asap so I added a quick turnaround for us.

Does this work?

I know that this only leaves a day for reviewing but it will be tight either way and there's not much we can do about it. 5/27 is a Thu, 5/28 is Fri.

Reply:
On May 29 I would want to tell any bursary applicants 
* if they got a bursary (travel + hotel) or just hotel,
* if they have been accepted (even if no bursary)

In the application form we should ask *(1) Will you attend only if you get travel + hotel? (2) Will you attend if you get hotel? (3) Will you attend without additional support?*

If we only informed bursary recipients that they have been accepted and not the category (3) (will attend regardless) then we make it difficult for them to book travel at reasonable cost.

Reply:
The registration form currently asks about attendance based on financial support in a slightly different way. After asking if people need support for hotel and/or travel, it asks estimated total costs (for travel) as well as amount of $ that could be obtained elsewhere (e.g., university travel awards, PI grants, etc.). It asks generally if they'd be unable to attend without the support asked for, so I think we could make decisions to answer the above.

Happy to change it to the way described above (or feel free to do so at https://docs.google.com/forms/d/1-_iVRym3oOxMkozB2TnBPWVUdk6ALJpAVt2fpNV9s8M/edit), but was just worried about people having fatigue answering so many questions

Reply:
The reg form looks good, thanks @jennaswa . I agree with you that it provides the information we need.